liqo-peering/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
I believe the commit message needs an explanation of what you are adding here and why. It needs to explain what this new liqo-peering chart does and why we need it.